packages/dx-chart-core/src/plugins/layout-manager/computeds.test.js

```diff
@@ -0,0 +1,28 @@
+import { bBoxes } from './computeds';
+
+describe('AxisCoordinates', () => {
+  it('should set new placeholder bBox', () => {
+    const result = bBoxes(
+      {},
+      { placeholder: 'placeholder', bBox: { width: 10, height: 20 } },
+    );
+    expect(result).toEqual({ placeholder: { width: 10, height: 20 } });
+  });
+
+  it('should change old bBox', () => {
+    const result = bBoxes(
+      { placeholder: { width: 10, height: 20 } },
+      { placeholder: 'placeholder', bBox: { width: 30, height: 40 } },
+    );
+    expect(result).toEqual({ placeholder: { width: 30, height: 40 } });
```

Comment:
Just a small note - usually, we [put](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-grid/src/grid.test.jsx#L42) matcher in the new line.

Reply:
Ok, it looks really nice